apps/server/src/modules/task-card/controller/api-test/task-card.api.spec.ts

```diff
@@ -72,14 +72,18 @@ describe('Task-Card Controller (api)', () => {
 
 		it('Create task-card should throw', async () => {
 			const user = setupUser([]);
+			const course = courseFactory.buildWithId({ teachers: [user] });
+			const inThreeDays = new Date(Date.now() + 259200000);
 
-			await em.persistAndFlush([user]);
+			await em.persistAndFlush([user, course]);
 			em.clear();
 
 			currentUser = mapUserToCurrentUser(user);
 
 			const params = {
 				title: 'title',
+				courseId: course.id,
+				dueDate: inThreeDays,
```

Comment:
so `visibleAtDate` is not required for now?

Reply:
@MartinSchuhmacher because it was marked as out of scope I ignored it. I think whole validation needs followUp-tickets